### PR TITLE
fix(Docker): fixes docker build issue

### LINKED ISSUE #2548: docker hub image out of date
Hi,

Is the automatic build of the [ohif/viewer](https://hub.docker.com/r/ohif/viewer) docker hub image broken?

I note that the last update to the "latest" tag was 5 months ago. but looking at the list of releases / commits to the github repo, there are many far more recent commits.

I was hoping to test #2466 which was merged recently.

Sure, I can build my own image, but I thought I'd raise an issue first, in case nobody was aware that the published images are out of date.

Thanks

### LINKED ISSUE #2709: Docker image build fails due to unsupported Python version
## Bug Report
Docker build image fails due to unsupported Python version 2.7
```
#30 581.2 gyp verb find Python - version is 2.7.16 - should be >=3.6.0
#30 581.2 gyp verb find Python - THIS VERSION OF PYTHON IS NOT SUPPORTED
```

### Describe the Bug
When I try to build the Docker image with `docker build .`, the instruction `yarn install` fails with error message

```
#30 581.2 gyp verb find Python - version is 2.7.16 - should be >=3.6.0
#30 581.2 gyp verb find Python - THIS VERSION OF PYTHON IS NOT SUPPORTED
```

### What steps can we follow to reproduce the bug?

1. Clone the repo
2. Run `docker build .`

```js
#30 581.2 gyp verb find Python Python is not set from command line or npm configuration
#30 581.2 gyp verb find Python Python is not set from environment variable PYTHON
#30 581.2 gyp verb find Python checking if "python3" can be used
#30 581.2 gyp verb find Python - executing "python3" to get executable path
#30 581.2 gyp verb find Python - "python3" is not in PATH or produced an error
#30 581.2 gyp verb find Python checking if "python" can be used
#30 581.2 gyp verb find Python - executing "python" to get executable path
#30 581.2 gyp verb find Python - executable path is "/usr/bin/python"
#30 581.2 gyp verb find Python - executing "/usr/bin/python" to get version
#30 581.2 gyp verb find Python - version is "2.7.16"
#30 581.2 gyp verb find Python - version is 2.7.16 - should be >=3.6.0
#30 581.2 gyp verb find Python - THIS VERSION OF PYTHON IS NOT SUPPORTED
#30 581.2 gyp ERR! find Python 
#30 581.2 gyp ERR! find Python Python is not set from command line or npm configuration
#30 581.2 gyp ERR! find Python Python is not set from environment variable PYTHON
#30 581.2 gyp ERR! find Python checking if "python3" can be used
#30 581.2 gyp ERR! find Python - "python3" is not in PATH or produced an error
#30 581.2 gyp ERR! find Python checking if "python" can be used
#30 581.2 gyp ERR! find Python - executable path is "/usr/bin/python"
#30 581.2 gyp ERR! find Python - version is "2.7.16"
#30 581.2 gyp ERR! find Python - version is 2.7.16 - should be >=3.6.0
#30 581.2 gyp ERR! find Python - THIS VERSION OF PYTHON IS NOT SUPPORTED
#30 581.2 gyp ERR! find Python 
#30 581.2 gyp ERR! find Python **********************************************************
#30 581.2 gyp ERR! find Python You need to install the latest version of Python.
#30 581.2 gyp ERR! find Python Node-gyp should be able to find and use Python. If not,
#30 581.2 gyp ERR! find Python you can try one of the following options:
#30 581.2 gyp ERR! find Python - Use the switch --python="/path/to/pythonexecutable"
#30 581.2 gyp ERR! find Python   (accepted by both node-gyp and npm)
#30 581.2 gyp ERR! find Python - Set the environment variable PYTHON
#30 581.2 gyp ERR! find Python - Set the npm configuration variable python:
#30 581.2 gyp ERR! find Python   npm config set python "/path/to/pythonexecutable"
#30 581.2 gyp ERR! find Python For more information consult the documentation at:
#30 581.2 gyp ERR! find Python https://github.com/nodejs/node-gyp#installation
#30 581.2 gyp ERR! find Python **********************************************************
#30 581.2 gyp ERR! find Python 
#30 581.2 gyp ERR! configure error 
#30 581.2 gyp ERR! stack Error: Could not find any Python installation to use
#30 581.2 gyp ERR! stack     at PythonFinder.fail (/usr/src/app/node_modules/node-gyp/lib/find-python.js:330:47)
#30 581.2 gyp ERR! stack     at PythonFinder.runChecks (/usr/src/app/node_modules/node-gyp/lib/find-python.js:159:21)
#30 581.2 gyp ERR! stack     at PythonFinder.<anonymous> (/usr/src/app/node_modules/node-gyp/lib/find-python.js:266:16)
#30 581.2 gyp ERR! stack     at PythonFinder.execFileCallback (/usr/src/app/node_modules/node-gyp/lib/find-python.js:297:7)
#30 581.2 gyp ERR! stack     at ChildProcess.exithandler (node:child_process:390:7)
#30 581.2 gyp ERR! stack     at ChildProcess.emit (node:events:520:28)
#30 581.2 gyp ERR! stack     at maybeClose (node:internal/child_process:1092:16)
#30 581.2 gyp ERR! stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:302:5)
#30 581.2 gyp ERR! System Linux 5.10.76-linuxkit
#30 581.2 gyp ERR! command "/usr/local/bin/node" "/usr/src/app/node_modules/node-gyp/bin/node-gyp.js" "rebuild" "--verbose" "--libsass_ext=" "--libsass_cflags=" "--libsass_ldflags=" "--libsass_library="
#30 581.2 gyp ERR! cwd /usr/src/app/node_modules/node-sass
#30 581.2 gyp ERR! node -v v16.14.0
#30 581.2 gyp ERR! node-gyp -v v8.4.1
#30 581.2 gyp ERR! not ok 
#30 581.2 Build failed with error code: 1
#30 581.2 info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
------
executor failed running [/bin/sh -c yarn install]: exit code: 1
```

### PR DESCRIPTION
**Pull request by pavankjadda**
Fixes the docker build issue caused by python 2.7x dependency. This PR addresses 3 issues

- Updates python version to python3 and fixes #2709 
- Switches Node JS version to LTS version i.e. 14
- Switches NgInx version to `stable-alpine` (LTS) instead of specific version 
- If merged this will also address #2548 

### Fixes
closes #2548 
fixes #2709

### PR Checklist

- [x] Brief description of changes
- [x] Links to any relevant issues
- [x] Required status checks are passing
- [x] User cases if changes impact the user's experience
- [x] `@mention` a maintainer to request a review

[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch

### CODE REVIEW

**Comment by pavankjadda** [comment]
@sedghi please review this

**Comment by swederik** [comment]
Thanks @pavankjadda! I've triggered the optional workflow on the CI to test docker publishing (https://app.circleci.com/pipelines/github/OHIF/Viewers/8852/workflows/5e15f550-de78-47d0-ac5b-7cb406e0b84c/jobs/18367) so we can see if that passes and if so we can merge it I think.

File: Dockerfile [hunk 1]
```diff
@@ -21,30 +21,30 @@
 
 # Stage 1: Build the application
 # docker build -t ohif/viewer:latest .
-FROM node:15.13.0-slim as builder
+FROM node:14-slim as builder
 
-RUN mkdir /usr/src/app
-WORKDIR /usr/src/app
```
Comment (pavankjadda) [inline Dockerfile:27]: Changed `/usr/src/app` to more standard `/app` directory

File: Dockerfile [hunk 2]
```diff
@@ -21,30 +21,30 @@
...
+COPY package.json /app/package.json
+COPY postcss.config.js /app/postcss.config.js
+COPY yarn.lock /app/yarn.lock
 
-RUN apt-get update && apt-get install -y python make g++
+RUN apt-get update && apt-get install -y python3.9 make g++
 # Run the install before copying the rest of the files
 RUN yarn config set workspaces-experimental true
-RUN yarn install
+RUN yarn install --frozen-lockfile
```
Comment (pavankjadda) [inline Dockerfile:45]: Installs dependencies from `yarn.lock` file rather from `package.json` file

File: Dockerfile [hunk 3]
```diff
@@ -53,14 +53,26 @@ RUN yarn run build
 
 # Stage 2: Bundle the built application into a Docker container
 # which runs Nginx using Alpine Linux
-FROM nginx:1.21.1-alpine
```
Comment (pavankjadda) [inline Dockerfile:56]: Switched to NgInx stable (LTS) alpine version

File: Dockerfile [hunk 4]
```diff
@@ -53,14 +53,26 @@ RUN yarn run build
...
+FROM nginx:alpine
 RUN apk add --no-cache bash
+
+# Delete existing Nginx configuration file
```
Comment (pavankjadda) [inline Dockerfile:59]: Added more comments on each step for better readability

**Comment by pavankjadda** [comment]
@swederik Please trigger the build now. Latest commits should fix the issue

**Comment by swederik** [comment]
This is really strange, I'm not sure why it failed. I'm guessing it's the downgrade in the Node version, or maybe (somehow) the yarn.lock usage? Any ideas?

```
@ohif/viewer: Hash: 5fd870cdb6c464a2da31
@ohif/viewer: Version: webpack 4.46.0
@ohif/viewer: Time: 172324ms
@ohif/viewer: Built at: 02/15/2022 4:28:51 PM
@ohif/viewer:                                                                                                                              Asset       Size  Chunks               Chunk Names
@ohif/viewer:                                                                                                         0.5fd870cdb6c464a2da31.css   4.44 KiB       0  [immutable]  ConnectedStandaloneRouting~IHEInvokeImageDisplay~StudyListRouting~ViewerLocalFileData~ViewerRouting
@ohif/viewer:                                                                                                         1.5fd870cdb6c464a2da31.css   2.32 KiB       1  [immutable]  ConnectedStandaloneRouting~IHEInvokeImageDisplay~ViewerLocalFileData~ViewerRouting
@ohif/viewer:                                                                                                        10.5fd870cdb6c464a2da31.css   14.2 KiB      10  [immutable]  vendors~app
@ohif/viewer:                                                                                                        12.5fd870cdb6c464a2da31.css   1.19 KiB      12  [immutable]  
@ohif/viewer:                                                                                                  12.bundle.d9d5eaec7d1b620ecc42.js     22 KiB      12  [immutable]  
@ohif/viewer:                                                                                                        13.5fd870cdb6c464a2da31.css  149 bytes      13  [immutable]  
@ohif/viewer:                                                                                                  13.bundle.f55d5d4ce0e5aca0739f.js   10.1 KiB      13  [immutable]  
@ohif/viewer:                                                                                                        14.5fd870cdb6c464a2da31.css  986 bytes      14  [immutable]  
@ohif/viewer:                                                                                                  14.bundle.cad4c4539c84e796e49f.js   13.9 KiB      14  [immutable]  
@ohif/viewer:                                                                                                  15.bundle.61dde0c30072c089d62f.js   16.7 KiB      15  [immutable]  
@ohif/viewer:                                                                                                  16.bundle.cd074a268b9d660a677f.js    984 KiB      16  [immutable]  
@ohif/viewer:                                                                                      16.bundle.cd074a268b9d660a677f.js.LICENSE.txt  795 bytes                       
@ohif/viewer:                                                                                                  17.bundle.58ce5130c8a6b254d172.js   4.45 KiB      17  [immutable]  
@ohif/viewer:                                                                                                  18.bundle.3f83abc1aa8eb3280b26.js   90.6 KiB      18  [immutable]  
@ohif/viewer:                                                                                      18.bundle.3f83abc1aa8eb3280b26.js.LICENSE.txt  192 bytes                       
@ohif/viewer:                                                                                                         5.5fd870cdb6c464a2da31.css   2.69 KiB       5  [immutable]  StudyListRouting
@ohif/viewer:                                                                                                         6.5fd870cdb6c464a2da31.css  271 bytes       6  [immutable]  ViewerLocalFileData
@ohif/viewer:                                                                                        CallbackPage.bundle.25e56ba7ad6ea29a845a.js   2.61 KiB       2  [immutable]  CallbackPage
@ohif/viewer:                                                                          ConnectedStandaloneRouting.bundle.64a388e1757206a58971.js   14.8 KiB       3  [immutable]  ConnectedStandaloneRouting
@ohif/viewer: ConnectedStandaloneRouting~IHEInvokeImageDisplay~StudyListRouting~ViewerLocalFileData~ViewerRouting.bundle.a6dd9ee76a6fbda95269.js   14.3 KiB       0  [immutable]  ConnectedStandaloneRouting~IHEInvokeImageDisplay~StudyListRouting~ViewerLocalFileData~ViewerRouting
@ohif/viewer:                  ConnectedStandaloneRouting~IHEInvokeImageDisplay~ViewerLocalFileData~ViewerRouting.bundle.a1a02be5bc4bb72449c4.js     39 KiB       1  [immutable]  ConnectedStandaloneRouting~IHEInvokeImageDisplay~ViewerLocalFileData~ViewerRouting
@ohif/viewer:                                                                               IHEInvokeImageDisplay.bundle.6f7a633540119e78cd52.js   7.98 KiB       4  [immutable]  IHEInvokeImageDisplay
@ohif/viewer:                                                                                    StudyListRouting.bundle.76db13e2ac1920210281.js   71.4 KiB       5  [immutable]  StudyListRouting
@ohif/viewer:                                                                                 ViewerLocalFileData.bundle.79508671571938d9c712.js   17.2 KiB       6  [immutable]  ViewerLocalFileData
@ohif/viewer:                                                                                       ViewerRouting.bundle.d249d3ec2526a7424234.js   15.6 KiB       7  [immutable]  ViewerRouting
@ohif/viewer:                                                                                                       app.5fd870cdb6c464a2da31.css    115 KiB       8  [immutable]  app
@ohif/viewer:                                                                                                 app.bundle.f8c461b0b44dbec6487a.js    772 KiB       8  [immutable]  app
@ohif/viewer:                                                                                                                              sw.js   1.92 KiB                       
@ohif/viewer:                                                                         vendors~ViewerLocalFileData.bundle.b23092b6647fd34bb039.js   13.9 KiB       9  [immutable]  vendors~ViewerLocalFileData
@ohif/viewer:                                                                                         vendors~app.bundle.0e7256eb7507b2706c1b.js   6.95 MiB      10  [immutable]  vendors~app
@ohif/viewer:                                                                             vendors~app.bundle.0e7256eb7507b2706c1b.js.LICENSE.txt    171 KiB                       
@ohif/viewer:                                                                     vendors~dicom-microscopy-viewer.bundle.a9c9d1a685e57a57b23f.js    728 KiB      11  [immutable]  vendors~dicom-microscopy-viewer
@ohif/viewer:                                                         vendors~dicom-microscopy-viewer.bundle.a9c9d1a685e57a57b23f.js.LICENSE.txt  173 bytes                       
@ohif/viewer: Entrypoint app = 10.5fd870cdb6c464a2da31.css vendors~app.bundle.0e7256eb7507b2706c1b.js app.5fd870cdb6c464a2da31.css app.bundle.f8c461b0b44dbec6487a.js
@ohif/viewer: ERROR in /app/node_modules/react-transition-group/esm/CSSTransition.js
@ohif/viewer: Module not found: Error: Can't resolve 'dom-helpers/addClass' in '/app/node_modules/react-transition-group/esm'
@ohif/viewer:  @ /app/node_modules/react-transition-group/esm/CSSTransition.js 5:0-47 13:11-22
@ohif/viewer:  @ /app/node_modules/react-transition-group/esm/index.js
@ohif/viewer:  @ ./OHIFStandaloneViewer.js
@ohif/viewer:  @ ./App.js
@ohif/viewer:  @ ./index.js
@ohif/viewer: ERROR in /app/node_modules/react-transition-group/esm/CSSTransition.js
@ohif/viewer: Module not found: Error: Can't resolve 'dom-helpers/removeClass' in '/app/node_modules/react-transition-group/esm'
@ohif/viewer:  @ /app/node_modules/react-transition-group/esm/CSSTransition.js 6:0-53 19:11-25
@ohif/viewer:  @ /app/node_modules/react-transition-group/esm/index.js
@ohif/viewer:  @ ./OHIFStandaloneViewer.js
@ohif/viewer:  @ ./App.js
@ohif/viewer:  @ ./index.js
@ohif/viewer: ERROR in /app/node_modules/pdfjs-dist/build/pdf.js 2094:26
@ohif/viewer: Module parse failed: Unexpected token (2094:26)
@ohif/viewer: You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
@ohif/viewer: |   async destroy() {
@ohif/viewer: |     this.destroyed = true;
@ohif/viewer: >     await this._transport?.destroy();
@ohif/viewer: |     this._transport = null;
@ohif/viewer: | 
@ohif/viewer:  @ /app/extensions/dicom-pdf/src/DicomPDFViewport.js 39:0-46 41:0-10
@ohif/viewer:  @ /app/extensions/dicom-pdf/src/OHIFDicomPDFViewport.js
@ohif/viewer:  @ /app/extensions/dicom-pdf/src/ConnectedOHIFDicomPDFViewer.js
@ohif/viewer:  @ /app/extensions/dicom-pdf/src/index.js
@ohif/viewer:  @ ./index.js
@ohif/viewer: error Command failed with exit code 1.
@ohif/viewer: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
lerna ERR! yarn run build:viewer exited 1 in '@ohif/viewer'
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
The command '/bin/sh -c yarn run build' returned a non-zero code: 1
```

**Comment by pavankjadda** [comment]
> This is really strange, I'm not sure why it failed. I'm guessing it's the downgrade in the Node version, or maybe (somehow) the yarn.lock usage? Any ideas?
> 
> ```
> @ohif/viewer: Hash: 5fd870cdb6c464a2da31
> @ohif/viewer: Version: webpack 4.46.0
> @ohif/viewer: Time: 172324ms
> @ohif/viewer: Built at: 02/15/2022 4:28:51 PM
> @ohif/viewer:                                                                                                                              Asset       Size  Chunks               Chunk Names
> @ohif/viewer:                                                                                                         0.5fd870cdb6c464a2da31.css   4.44 KiB       0  [immutable]  ConnectedStandaloneRouting~IHEInvokeImageDisplay~StudyListRouting~ViewerLocalFileData~ViewerRouting
> @ohif/viewer:                                                                                                         1.5fd870cdb6c464a2da31.css   2.32 KiB       1  [immutable]  ConnectedStandaloneRouting~IHEInvokeImageDisplay~ViewerLocalFileData~ViewerRouting
> @ohif/viewer:                                                                                                        10.5fd870cdb6c464a2da31.css   14.2 KiB      10  [immutable]  vendors~app
> @ohif/viewer:                                                                                                        12.5fd870cdb6c464a2da31.css   1.19 KiB      12  [immutable]  
> @ohif/viewer:                                                                                                  12.bundle.d9d5eaec7d1b620ecc42.js     22 KiB      12  [immutable]  
> @ohif/viewer:                                                                                                        13.5fd870cdb6c464a2da31.css  149 bytes      13  [immutable]  
> @ohif/viewer:                                                                                                  13.bundle.f55d5d4ce0e5aca0739f.js   10.1 KiB      13  [immutable]  
> @ohif/viewer:                                                                                                        14.5fd870cdb6c464a2da31.css  986 bytes      14  [immutable]  
> @ohif/viewer:                                                                                                  14.bundle.cad4c4539c84e796e49f.js   13.9 KiB      14  [immutable]  
> @ohif/viewer:                                                                                                  15.bundle.61dde0c30072c089d62f.js   16.7 KiB      15  [immutable]  
> @ohif/viewer:                                                                                                  16.bundle.cd074a268b9d660a677f.js    984 KiB      16  [immutable]  
> @ohif/viewer:                                                                                      16.bundle.cd074a268b9d660a677f.js.LICENSE.txt  795 bytes                       
> @ohif/viewer:                                                                                                  17.bundle.58ce5130c8a6b254d172.js   4.45 KiB      17  [immutable]  
> @ohif/viewer:                                                                                                  18.bundle.3f83abc1aa8eb3280b26.js   90.6 KiB      18  [immutable]  
> @ohif/viewer:                                                                                      18.bundle.3f83abc1aa8eb3280b26.js.LICENSE.txt  192 bytes                       
> @ohif/viewer:                                                                                                         5.5fd870cdb6c464a2da31.css   2.69 KiB       5  [immutable]  StudyListRouting
> @ohif/viewer:                                                                                                         6.5fd870cdb6c464a2da31.css  271 bytes       6  [immutable]  ViewerLocalFileData
> @ohif/viewer:                                                                                        CallbackPage.bundle.25e56ba7ad6ea29a845a.js   2.61 KiB       2  [immutable]  CallbackPage
> @ohif/viewer:                                                                          ConnectedStandaloneRouting.bundle.64a388e1757206a58971.js   14.8 KiB       3  [immutable]  ConnectedStandaloneRouting
> @ohif/viewer: ConnectedStandaloneRouting~IHEInvokeImageDisplay~StudyListRouting~ViewerLocalFileData~ViewerRouting.bundle.a6dd9ee76a6fbda95269.js   14.3 KiB       0  [immutable]  ConnectedStandaloneRouting~IHEInvokeImageDisplay~StudyListRouting~ViewerLocalFileData~ViewerRouting
> @ohif/viewer:                  ConnectedStandaloneRouting~IHEInvokeImageDisplay~ViewerLocalFileData~ViewerRouting.bundle.a1a02be5bc4bb72449c4.js     39 KiB       1  [immutable]  ConnectedStandaloneRouting~IHEInvokeImageDisplay~ViewerLocalFileData~ViewerRouting
> @ohif/viewer:                                                                               IHEInvokeImageDisplay.bundle.6f7a633540119e78cd52.js   7.98 KiB       4  [immutable]  IHEInvokeImageDisplay
> @ohif/viewer:                                                                                    StudyListRouting.bundle.76db13e2ac1920210281.js   71.4 KiB       5  [immutable]  StudyListRouting
> @ohif/viewer:                                                                                 ViewerLocalFileData.bundle.79508671571938d9c712.js   17.2 KiB       6  [immutable]  ViewerLocalFileData
> @ohif/viewer:                                                                                       ViewerRouting.bundle.d249d3ec2526a7424234.js   15.6 KiB       7  [immutable]  ViewerRouting
> @ohif/viewer:                                                                                                       app.5fd870cdb6c464a2da31.css    115 KiB       8  [immutable]  app
> @ohif/viewer:                                                                                                 app.bundle.f8c461b0b44dbec6487a.js    772 KiB       8  [immutable]  app
> @ohif/viewer:                                                                                                                              sw.js   1.92 KiB                       
> @ohif/viewer:                                                                         vendors~ViewerLocalFileData.bundle.b23092b6647fd34bb039.js   13.9 KiB       9  [immutable]  vendors~ViewerLocalFileData
> @ohif/viewer:                                                                                         vendors~app.bundle.0e7256eb7507b2706c1b.js   6.95 MiB      10  [immutable]  vendors~app
> @ohif/viewer:                                                                             vendors~app.bundle.0e7256eb7507b2706c1b.js.LICENSE.txt    171 KiB                       
> @ohif/viewer:                                                                     vendors~dicom-microscopy-viewer.bundle.a9c9d1a685e57a57b23f.js    728 KiB      11  [immutable]  vendors~dicom-microscopy-viewer
> @ohif/viewer:                                                         vendors~dicom-microscopy-viewer.bundle.a9c9d1a685e57a57b23f.js.LICENSE.txt  173 bytes                       
> @ohif/viewer: Entrypoint app = 10.5fd870cdb6c464a2da31.css vendors~app.bundle.0e7256eb7507b2706c1b.js app.5fd870cdb6c464a2da31.css app.bundle.f8c461b0b44dbec6487a.js
> @ohif/viewer: ERROR in /app/node_modules/react-transition-group/esm/CSSTransition.js
> @ohif/viewer: Module not found: Error: Can't resolve 'dom-helpers/addClass' in '/app/node_modules/react-transition-group/esm'
> @ohif/viewer:  @ /app/node_modules/react-transition-group/esm/CSSTransition.js 5:0-47 13:11-22
> @ohif/viewer:  @ /app/node_modules/react-transition-group/esm/index.js
> @ohif/viewer:  @ ./OHIFStandaloneViewer.js
> @ohif/viewer:  @ ./App.js
> @ohif/viewer:  @ ./index.js
> @ohif/viewer: ERROR in /app/node_modules/react-transition-group/esm/CSSTransition.js
> @ohif/viewer: Module not found: Error: Can't resolve 'dom-helpers/removeClass' in '/app/node_modules/react-transition-group/esm'
> @ohif/viewer:  @ /app/node_modules/react-transition-group/esm/CSSTransition.js 6:0-53 19:11-25
> @ohif/viewer:  @ /app/node_modules/react-transition-group/esm/index.js
> @ohif/viewer:  @ ./OHIFStandaloneViewer.js
> @ohif/viewer:  @ ./App.js
> @ohif/viewer:  @ ./index.js
> @ohif/viewer: ERROR in /app/node_modules/pdfjs-dist/build/pdf.js 2094:26
> @ohif/viewer: Module parse failed: Unexpected token (2094:26)
> @ohif/viewer: You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
> @ohif/viewer: |   async destroy() {
> @ohif/viewer: |     this.destroyed = true;
> @ohif/viewer: >     await this._transport?.destroy();
> @ohif/viewer: |     this._transport = null;
> @ohif/viewer: | 
> @ohif/viewer:  @ /app/extensions/dicom-pdf/src/DicomPDFViewport.js 39:0-46 41:0-10
> @ohif/viewer:  @ /app/extensions/dicom-pdf/src/OHIFDicomPDFViewport.js
> @ohif/viewer:  @ /app/extensions/dicom-pdf/src/ConnectedOHIFDicomPDFViewer.js
> @ohif/viewer:  @ /app/extensions/dicom-pdf/src/index.js
> @ohif/viewer:  @ ./index.js
> @ohif/viewer: error Command failed with exit code 1.
> @ohif/viewer: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
> lerna ERR! yarn run build:viewer exited 1 in '@ohif/viewer'
> error Command failed with exit code 1.
> info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
> The command '/bin/sh -c yarn run build' returned a non-zero code: 1
> ```

Let me try Node version 16 LTS

**Comment by wayfarer3130** [comment]
I'd also suggest removing the docz code, but leave the commenting in and
just add instructions on how it could be manually run if it is installed
locally.  Perhaps commit a copy of the output documentation as existing
reference docs.

Bill

**Comment by ladeirarodolfo** [comment]
@pavankjadda  thanks for your support, your PR fixes the current package incompatibility and docker build. LGTM

@swederik  removing docz would not fix it in totally but only few issues related to package versioning. I will open a PR to remove docz

**Review by ladeirarodolfo — APPROVED** [review]
LGTM

**Comment by pavankjadda** [comment]
> We just merged @ladeirarodolfo 's work to fix this. Please let me know if you have any issues with that change. Thanks so much for your work on the issue! I'll close this because I think it's redundant now.

No problem.

**Pull request closed without merging**